packages/atoms/src/NumPadButton.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Pressable, PressableProps } from 'react-native';
+
+import { Text } from './Text';
+import { useNativeStyles, NativeStyleObject, prepareNativeStyle } from '@trezor/styles';
+
+export interface NumPadButtonProps extends Omit<PressableProps, 'style'> {
+    title: string;
+    onPress: () => void;
```

Comment:
`onPress` should provide `value` aka: `onPress: (value: number) => void` so you can use this component like this:

```tsx
const handlePress = (value: number) => {
	console.log(value);
}
<NumPadButton value={42} onPress={handlePress} />
```

and it will log 42 when pressed